lethal-squirrels/Bangazon-Site
# Chaz nav bar

## REFERENCE

Ticket: (#28)

## DESCRIPTION
Added nav bar links so they go to the correct places. 
At the moment, a user being logged in does not change what is displayed. That will come in a later ticket. 

## STEPS TO TEST
1.  Press `Ctrl + f5` to build the project.
1. Click on each of the links and make sure they direct you to their respective pages
1. Let me know if I missed any.
